### DISCUSSION TITLE
Error codes (washing machine)

### DISCUSSION THREAD

**Discussion by Aephir** [astrandb/miele#410] [Q&A] [answered]
I have a washing machine, and can see the `binary_sensor.washing_machine_failure`, that I assume just turns `on` when there is. Is the actual error code reported in any of the provided entities? And if so, is there a list somewhere of error codes available in the integration?

Basically, I have notifications set up. Since some of the kids are starting to help with light laundry duties, I would like to be able to parse a "what to do" in the notification text based on what the error is. The utopian hope is that they might be able to follow instructions, while I know they'd never google the error code :)

But as this is a device I installed yesterday, I'm assuming (hoping) it will be a while until I see my first error code, so I don't want to wait for that before setting this up.

**Comment by mundschenk-at** [comment, marked as answer]
No, the Miele 3rd-Party API does not include any information on the specific errors.

**Reply by Aephir** [reply]
Too bad, but thanks for confirming!
